csl_impedance_controller:
    arm_id: panda
    type: csl_controllers/CSLImpedanceController
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7